aries-site/src/tests/accessibility/axe.js

```diff
@@ -25,7 +25,10 @@ test('should check Home page', async t => {
   if (t.browser.name === 'Chrome') {
     const axeContext = {
       // excluding the HPE Slim header that gets a complaint about link text
-      exclude: ['.hpehf-close-btn'],
+      exclude: [
+        '.hpehf-close-btn',
+        '.hpehf-help-and-support > .hpehf-help-menu-dropdown-list',
```

Comment:
Need to look into this further -- tried excluding these elements from global header bc it's causing accessibility failure but it didn't seem to resolve.